notebooks/QA_extraction_generation.ipynb

```diff
@@ -0,0 +1,556 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.                                   max_length=100) #Increasing the length
</pre><p>Is there an auto length deciding variant available? Or in other words, can it know when to stop on it's own?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>